portfolio/pom.xml

```diff
@@ -23,8 +23,32 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-1.0-sdk</artifactId>
+      <version>1.9.59</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.6</version>
+    </dependency>
+
+    <dependency>
```

Comment:
why duplicate the maven dependency?

Reply:
I'm not too sure I guess I just hit copy paste twice

Reply:
![Dependecies](https://user-images.githubusercontent.com/27080061/85788161-235f2400-b6f2-11ea-976f-53fb1cf35331.JPG)